service/src/test/resources/db/migration/V1472923547__configs_create_table_client_systemflags.sql

```diff
@@ -0,0 +1,45 @@
+USE configs;
```

Comment:
I know these are only used in tests, but I think we should add comments at the top of each migration file explaining why the tables are being built/altered.  That'll help us and other contributors see why we made the decisions we did and also will help troubleshooting in the future.